README.md

```diff
@@ -1,79 +1,73 @@
+    <a href="https://x.com/next_hat" target="_blank"><b>𝕏</b></a>
   </p>
   <p>
 
 [![Tests](https://github.com/next-hat/nanocl/actions/workflows/tests.yml/badge.svg)](https://github.com/next-hat/nanocl/actions/workflows/tests.yml)
 [![Clippy](https://github.com/next-hat/nanocl/actions/workflows/clippy.yml/badge.svg)](https://github.com/next-hat/nanocl/actions/workflows/clippy.yml)
-
-  </p>
-
-  <p>
-
 [![codecov](https://codecov.io/gh/next-hat/nanocl/branch/nightly/graph/badge.svg?token=4I60HOW6HM)](https://codecov.io/gh/next-hat/nanocl)
 
   </p>
-
 </div>
 
+**Nanocl** is an open source distributed system designed to revolutionize **cloud native** from the ground up.
+
+The developing ecosystem provides an **easy** solution for **local** development, avoiding common issues like CORS & cookies when working with **complex microservices**.
+
+The architecture of Nanocl enables a **seamless deployment** to **production** among other innovations.<br/>
 <blockquote>
  <span>
-   Test, Deploy, Scale, Monitor, Orchestrate
+    You could build a K8s within Nanocl. But we are quite sure you wouldn't want to..
  </span>
 </blockquote>
 
+Nanocl will introduce the **next paradigm** for **Platform Engineers** by providing ground breaking cloud-native architectural innovations, best-in-class security, while reducing operational costs with e.g. 🦀 **Rust** based efficiency.
 
-**Nanocl** is an open-source distributed system designed to revolutionize developers' workflows.<br/>
-It aims to provide an easy solution for localhost development, addressing common issues like CORS and cookies when working with complex microservice architectures.<br/>
-Nanocl also facilitates seamless deployment to production.<br/>
-By empowering developers and DevOps professionals, Nanocl helps optimize workflows, enhance security, and reduce operational costs.
-
-**Join our *[discord][discord]* and help us shape the future of infrastructure management.**
-
-## Latest news
-
-- [2024/11/24] [Automating deployment with GitHub Actions](https://docs.next-hat.com/blog/automating-deployment-with-github-actions-and-nanocl)
-- [2024/11/01] [End to End TLS encryption and first step for network meshing](https://docs.next-hat.com/blog/nanocl-0.16) (release)
-- [2024/06/11] [Man page, Backup, Remove Orphans and more](https://docs.next-hat.com/blog/nanocl-0.15) (release)
-- [2024/06/01] [We are invited to the Merge Berlin 2024](https://www.linkedin.com/feed/update/urn:li:activity:7201921660289998850) (event)
-- [2024/05/07] [Context, SubState and more](https://docs.next-hat.com/blog/nanocl-0.14) (release)
+<blockquote>
+ <span>
+    Who said that K8s is more than a perfect platform for application workloads?
+ </span>
+</blockquote>
 
 ## Installation
 
-To install **Nanocl** for your system, please refer to our online [installation guide][nanocl_install_guide].
+We are already compatible with **Debian/Ubuntu, MacOS & Windows**, just jump to [Installation guide][nanocl_install_guide].<br/>
```

Comment:
Can you replace Debian/Ubuntu with linux ?

Reply:
This has to be corrected within the documentation, too.

Reply:
Yeap will do during the day

Reply:
Perfect. Leo is a real hustler.